### ISSUE TITLE
Shift-clicking doesn't remove search box focus

### ISSUE THREAD

**Issue by scribbler957** [refinedmods/refinedstorage#2913]
When moving items by clicking the search focus is removed. When you move them by shift-click, the focus stays. I think the clicking behavior should be consistent. Right now I shift-click out and immediately hit 'e' and instead of closing, the 'e' is added to my search, which isn't the desired behavior.

MC 1.16.5
RS 1.9.12
Enigmatica 6 modpack 0.4.3

**Comment by TemperedFool** [comment]
Agreed - I run into this constantly when trying to leave a terminal. I know what I need to do but my fingers refuse.

I'd go so far as saying clicking anywhere else in the GUI should take focus away from the search field.

**Comment by GoldenretriverYT** [comment]
agreed, its by far the most annoying difference compared to AE2

**Comment by raoulvdberge** [comment]
> When moving items by clicking the search focus is removed.  

This is no longer true in latest RS (v1.12.3). If I click while it's selected, the selection stays.

> When you move them by shift-click, the focus stays. 

This is true.

The behavior is at least now consistent.

But we should probably defocus now.
